cylinder_radius: 1
cylinder_width: 1
dof: 6
measures_ratio: 0.5
maximum_iter: 30000
multigoal:
  use_kdtree: false
  type: Multigoal
  max_distance: 0.5
  max_refine_time: 3.0   # time for refine solution
  rewire_radius: 2.0
  extend: true
  local_bias: 0.25
  forgetting_factor: 0.999
  tube_radius: 0.02
  utopia_tolerance: 0.05  # exit condition for early stop (cost< (1+utopia_tolerance) * distance)
  warp: false
  warp_once: true
  collision_distance: 0.004
  reward_fcn: "pippo"
